students/WesleyM/Lesson2/Print_Grid.py

```diff
@@ -0,0 +1,36 @@
+#prints a 2x2 grid
+def grid(): 
+    horz = ('+'+'----')*2+'+'
+    vert = ('|'+'    ')*2+'|'
+    for i in range(1,12):
+        if i == 1 or i==6 or i==11:
+            print(horz)
+        else:
+            print(vert)
+    return
```

Comment:
`return` is not required in python functions; since your functions don't return a value you can omit the return statement